app/helpers/react_components/editor.rb

```diff
@@ -32,7 +32,8 @@ def to_s
           },
           track: {
             title: track.title,
-            slug: track.slug
+            slug: track.slug,
+            help_html: Markdown::Parse.(track.git.help)
```

Comment:
What we actually want to show is an interesting question. At the least, we want to show the `exercises/shared/.docs/help.md` file, but we could also show the more general help information that we append to the `HELP.md` when an exercise is downloaded via the CLI.

For example, this is what the corresponding `HELP.md` section looks like for a C# exercise:

```markdown
## Need to get help?

If you'd like help solving the exercise, check the following pages:

- The [C# track's documentation](https://exercism.org/docs/tracks/csharp)
- [Exercism's support channel on gitter](https://gitter.im/exercism/support)
- The [Frequently Asked Questions](https://exercism.org/docs/using/faqs)

Should those resources not suffice, you could submit your (incomplete) solution to request mentoring.

To get help if you're having trouble, you can use one of the following resources:

- [Gitter](https://gitter.im/exercism/xcsharp) is Exercism C# track's Gitter room; go here to get support and ask questions related to the C# track.
- [/r/csharp](https://www.reddit.com/r/csharp) is the C# subreddit.
- [StackOverflow](http://stackoverflow.com/questions/tagged/c%23) can be used to search for your problem and see if it has been answered already. You can also ask and answer questions.
```

What's interesting is that we show the general hints first, whereas I think we should actually be showing the track-specific hints first. Thoughts?

Reply:
Okay, I've updated. This still needs some UI work, getting its own icon.